ktor-client/ktor-client-plugins/ktor-client-auth/common/src/io/ktor/client/plugins/auth/Auth.kt

```diff
@@ -128,14 +130,14 @@ public val Auth: ClientPlugin<AuthConfig> = createClientPlugin("Auth", ::AuthCon
 
     on(Send) { originalRequest ->
         val origin = proceed(originalRequest)
-        if (origin.response.status != HttpStatusCode.Unauthorized) return@on origin
+        if (origin.response.status !in pluginConfig.reAuthStatusCodes) return@on origin
         if (origin.request.attributes.contains(AuthCircuitBreaker)) return@on origin
 
         var call = origin
 
         val candidateProviders = HashSet(providers)
 
-        while (call.response.status == HttpStatusCode.Unauthorized) {
+        while (call.response.status in pluginConfig.reAuthStatusCodes) {
             LOGGER.trace("Received 401 for ${call.request.url}")
```

Comment:
Could you also change log messages to reflect the changes and replace `.trace(...)` call with `.trace { ... }` in this file?

Reply:
Sure. Done in the isUnauthorized commit.